### PR TITLE
Add disconnected service doc and diagram

### LINKED ISSUE #157: Document hierarchical port binding / disconnected services
#### OpenStack Release
Liberty +

#### Description
New documentation is required for the new hierarchical port binding feature. The documentation should be created on / merged into the feature's development branch. It should utilize the standard docs feature template.

### PR DESCRIPTION
**Pull request by jputrino**
@mattgreene @jlongstaf @jgruber 

#### What issues does this address?
Fixes #157

#### What's this change do?
Adds documentation for hierarchical port binding.

#### Where should the reviewer start?

View the docs in the test site at http://lbaasv2-test.readthedocs.io/en/feature.disconnected-service/includes/topic_hierarchical-port-binding.html. 

#### Any background context?

Note to self: Once this branch is merged into Liberty, I'll need to add this doc to the TOC and update refs to other feature docs (if needed).

### CODE REVIEW

File: docs/includes/topic_hierarchical-port-binding.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+.. figure:: ../media/lbaasv2_hierarchical-port-binding.png
+    :alt: F5 LBaaSv2 Hierarchical Port Binding
+
+    F5 LBaaSv2 Hierarchical Port Binding
+
+
+Prerequisites
+-------------
+
+- Licensed, operational BIG-IP :term:`device` or :term:`device cluster`.
```
Comment (mattgreene) [inline docs/includes/topic_hierarchical-port-binding.rst]: Should this change to reflect recent learnings?  Perhaps:
Operational BIG-IP device or device cluster, licensed for "SDN Services"

File: docs/includes/topic_hierarchical-port-binding.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
+Hierarchical Port Binding
+=========================
+
+Overview
+--------
+
+Neutron `hierarchical port binding`_ [#]_ allows software-defined networking (SDN) users to dynamically configure VLANs and VLAN tags for a physical BIG-IP® :term:`device` or :term`device service cluster` connected to a 'top of rack' L3 switch (a network 'segment'). Telling the F5® agent what physical switch and port the BIG-IPs are connected to allows the agent to configure the BIG-IPs to process traffic for VLANs that are dynamically created in that segment.
```
Comment (mattgreene) [inline docs/includes/topic_hierarchical-port-binding.rst]: The SDN can potentially consist of all network types (VLAN, VXLAN and GRE), not just VLAN.

File: docs/includes/topic_hierarchical-port-binding.rst [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+- Knowledge of `OpenStack Networking <http://docs.openstack.org/liberty/networking-guide/>`_ concepts.
+
+- Knowledge of BIG-IP `system configuration`_, `local traffic management`_, & `device service clustering`_.
+
+
+Caveats
+-------
+
+- In release v |release| of the F5 LBaaSv2 driver and agent, ``VLAN`` is the only supported ML2 network type.
```
Comment (mattgreene) [inline docs/includes/topic_hierarchical-port-binding.rst]: Where did this come from?  We've been testing VLAN, VXLAN and GRE since the MVP release.

Comment (jputrino) [inline docs/includes/topic_hierarchical-port-binding.rst]: This info came from @jgruber. I can certainly change it if it's wrong!

File: docs/includes/topic_hierarchical-port-binding.rst [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    |                                      || See :ref:`disconnected services`.        |               |
+    +--------------------------------------+-------------------------------------------+---------------+
+    |f5_network_segment_gross_timeout      || Maximum seconds to wait for a network to | 300           |
+    |                                      || be bound before the LBaaS request fails. |               |
+    |                                      || See :ref:`disconnected services`.        |               |
+    +--------------------------------------+-------------------------------------------+---------------+
+
+
+
+.. topic:: Example
```
Comment (mattgreene) [inline docs/includes/topic_hierarchical-port-binding.rst]: Suggest the example show a customer how to configure settings if they want to run in "disconnected service" mode.  Uncomment the 3 settings, and supply sample text.

File: docs/includes/topic_hierarchical-port-binding.rst (hunk 1 above)
Comment (jputrino) [inline docs/includes/topic_hierarchical-port-binding.rst]: Definitely.

File: docs/includes/topic_hierarchical-port-binding.rst [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+
+2. Configure the heirarchical port binding settings in the :ref:`L2/L3 Segmentation Modes` section of the config file.
+
+.. table:: Hierarchical Port Binding settings
+
+    +--------------------------------------+-------------------------------------------+---------------+
+    | Setting                              | Description                               | Default Value |
+    +======================================+===========================================+===============+
+    |f5_network_segment_physical_network   || The name of the network segment in which | None          |
```
Comment (mattgreene) [inline docs/includes/topic_hierarchical-port-binding.rst]: Please elaborate in the description for this first setting.  The setting MUST be commented out to run in "traditional mode" (not disconnected).  The setting MUST be uncommented and set to a valid network name to run in "disconnected mode".  We can also fix the comment block in the ini file, but I'm trusting customers to use these new docs to understand the feature.

**Comment by mattgreene** [comment]
Done with review.

File: docs/includes/topic_hierarchical-port-binding.rst (hunk 3 above)
Comment (mattgreene) [inline docs/includes/topic_hierarchical-port-binding.rst]: Ok.  Offline discussion clarified this part.  Please amend the caveat to state something like "... is the only supported ML2 network type when employing Hierarchical Port Binding."

Comment (jputrino) [inline docs/includes/topic_hierarchical-port-binding.rst]: done.

File: docs/includes/topic_hierarchical-port-binding.rst (hunk 2 above)
Comment (jputrino) [inline docs/includes/topic_hierarchical-port-binding.rst]: fixed

File: docs/includes/topic_hierarchical-port-binding.rst (hunk 4 above)
Comment (jputrino) [inline docs/includes/topic_hierarchical-port-binding.rst]: done.

File: docs/includes/topic_hierarchical-port-binding.rst (hunk 5 above)
Comment (jputrino) [inline docs/includes/topic_hierarchical-port-binding.rst]: done.

**Comment by jputrino** [comment]
All set! Fixes are incorporated in the staged docs: http://lbaasv2-test.readthedocs.io/en/feature.disconnected-service/includes/topic_hierarchical-port-binding.html

**Pull request merged**